src/pages/balances/nfts.tsx

```diff
@@ -12,18 +12,18 @@ import { useRemoteSafeApps } from '@/hooks/safe-apps/useRemoteSafeApps'
 const NftApps = memo(function NftApps(): ReactElement | null {
   const [nftApps] = useRemoteSafeApps(SafeAppsTag.NFT)
 
-  if (!nftApps?.length) {
+  if (nftApps?.length === 0) {
     return null
   }
 
   return (
     <Grid item sm={12} lg={3} order={{ lg: 1 }}>
-      <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2} mt={0.6}>
+      <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2} mt={0.75}>
```

Comment:
Rather put pixels here no?

Reply:
It will scale with the rest of the grid if change the base values.